osf/metrics.py

```diff
@@ -289,8 +283,7 @@ def get_current_user_metrics(cls, institution) -> list:
         """
         last_record_time = cls.get_recent_datetime(institution)
         search = cls.filter_institution(institution).sort('timestamp')
-
-        search = search.filter('range', timestamp={'gte': last_record_time})
+        search = search.filter('range', timestamp={'gte': last_record_time}).extra(size=settings.MAX_SIZE_OF_ES_QUERY)
```

Comment:
This was actually the cause of the "pagination" error this defaults to a size of ten, which is the same as the pagination default.